### ISSUE TITLE
Evaluate the recommendation system

### ISSUE THREAD

**Issue by AnhPC03** [krishnaik06/Recommendation_complete_tutorial#1]
How can i evaluate the book recommandation system?
Must i split the dataset into train and val or test set?

**Comment by Dario-Farina-95** [comment]
i don't think you should split the data set. i think that just eliminate the first recommandation should be enough, the more data you have the better the model should performe. in this case the classical "overfitting" problem is a good thing , you want the most specific racommandation for that particular movie.

for testing the recommandation system you should have a set of movie with a predetermine racommandation and maybe watch how similar the "human racommandation" and the knn racommandatio are, but i think that this is very impractical
